examples/demo-jobs/.neuro/upload-local-action.yml

```diff
@@ -0,0 +1,11 @@
+kind: local
+name: Upload files to storage
+author: Roman Skirikhin
+descr: Uploads files to neuro storage
+inputs:
+  from:
+    descr: Local path
+  to:
+    descr: "Path on storage. Should start with 'storage:'"
+shell:
```

Comment:
Please replace `shell` with `cmd` here and below, the action doesn't start a shell technically.